modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/unified_query_builder/ScopeSelector.tsx

```diff
@@ -89,7 +90,12 @@ export default function ScopeSelector({
 							selectedItem.groupexternalreferencecode,
 						name: selectedItem.groupdescriptivename,
 						status: STATUS.ACTIVE,
-						type: selectedItem.groupscopelabel,
+						type:
+							selectedItem.groupdepotentrytype === '1'
+								? Liferay.Language.get('space')
+								: selectedItem.groupdepotentrytype === '0'
+									? Liferay.Language.get('asset-library')
+									: Liferay.Language.get('site'),
```

Comment:
It seems strange having display logic while creating a data object. It'd reduce the flexibility like if you wanted to filter by type or something like that.

This could be:
```
type: selectedItem.groupdepotentrytype
```

And then you display logic would go directly at the part it's rendering: https://github.com/oliv-yu/liferay-portal/blob/a175f3a123806ee5ff05d6cdad8d40fe40b924d3/modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/unified_query_builder/ScopeSelector.tsx#L187

(Assuming that's the only place this type is used)

Reply:
Thank you, that worked really well! Included it as the last commit 👍